honeybadgermpc/hbavss.py

```diff
@@ -14,9 +14,12 @@
 
 class HbAVSSMessageType:
     OK = "OK"
+    IMPLICATE = "IMPLICATE"
+    READY = "READY"
 
 
-class HbAvssLight(object):
```

Comment:
We want to keep `HbAvssLight` available as well. It seems like this patch is getting rid of HbAvssLight altogether, but we still want both. My preference will be to have this as a separate file `hbavss.py` and `hbavss_light.py` both available

Reply:
It is not removed. The light version is still there. I made a base class containing both classes and they are all in the same file ``hbavss.py``

Reply:
OK I misread, marking as resolved